views/errors.slim

```diff
@@ -36,5 +36,10 @@ html
                 tr
                   td = error.heading + 2
                   td = field_name
-                  td = message.join(' AND ')
+                  - if message.is_a?(Hash)
```

Comment:
Yeah, this slim definitely needs a refactor. I wanted to get the tests green, then raise another card to split this slim up and fix all of this. 

I didn't realise is_a is a smell. Thanks for that, will look to fix it soon.